### PR TITLE
Upgrade to play 2.8.x and Scala 2.13.

### PR DESCRIPTION
**Pull request by ziemek**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -1,35 +1,28 @@
-
 organization := "com.kashoo"
-
 name := """ws-limited"""
-
-version := "1.1.0"
-
+version := "4.0.0"
```
Comment (dkichler) [inline build.sbt:3]: Why such a big jump?

File: app/com/kashoo/ws/RequestRateLimits.scala [hunk 2]
```diff
@@ -41,13 +44,13 @@ object RequestRateLimits {
 
   def apply(rrl: RequestRateLimit*): RequestRateLimits = Set[RequestRateLimit](rrl:_*)
 
-  def apply(config: Configuration): RequestRateLimits = {
-    val rateLimiterEc = config.getConfig("ws.limited.execution-context") match {
-      case Some(ecConfig) => Akka.system.dispatchers.lookup("ws.limited.execution-context")
-      case None => play.api.libs.concurrent.Execution.Implicits.defaultContext
+  def apply(config: Configuration, actorSystem: ActorSystem)(implicit ec: ExecutionContext): RequestRateLimits = {
```
Comment (dkichler) [inline app/com/kashoo/ws/RequestRateLimits.scala]: it's not uncommon to have `actorSystem` be an implicit parameter, similar to `ec: ExecutionContext`

File: app/com/kashoo/ws/WSLimitedClientProvider.scala [hunk 3]
```diff
@@ -1,19 +1,21 @@
...
 
 /**
   * Provider that wraps clients provided by WSAPI with [[com.kashoo.ws.WSLimitedClientAdapter]]
   *
   * @param config application configuration
-  * @param wsApi WSAPI implementation
+  * @param ws WSClient implementation
   */
-class WSLimitedClientProvider @Inject() (config: Configuration, wsApi: WSAPI) extends Provider[WSClient] {
+class WSLimitedClientProvider @Inject() (config: Configuration, ws: WSClient, actorSystem: ActorSystem)(implicit ec: ExecutionContext) extends Provider[WSClient] {
```
Comment (dkichler) [inline app/com/kashoo/ws/WSLimitedClientProvider.scala]: same here

**Review by dkichler — APPROVED** [review]
👍 👍 - left a question and a comment but looks good

File: build.sbt (hunk 1 above)
Comment (ziemek) [reply build.sbt:3]: It looks funny out of context and maybe I should change it to 2.0.0. The reason was that versions 2 and 3 were used by play 2.6 and 2.7, though not merged into master: https://github.com/Kashoo/ws-limited/blob/play-2.7.x/build.sbt.

File: app/com/kashoo/ws/WSLimitedClientProvider.scala (hunk 3 above)
Comment (ziemek) [reply app/com/kashoo/ws/WSLimitedClientProvider.scala]: 👍  I made both ActorSystem parameters implicit.

**Review by dkichler — APPROVED** [review]
👍

**Comment by ziemek** [comment]
Merging based on Dave's behalf.

**Pull request merged**